gef.py

```diff
@@ -1523,6 +1539,15 @@ def is_call(self, insn):
         call_mnemos = {"call", "callq"}
         return mnemo in call_mnemos
 
+    @property
+    def get_stack_top(self):
+        return get_register("$ebp")
+
+    @property
+    def sp(self):
+        return get_register("$esp")
```

Comment:
Based on your comments, I'm doing this in a more generic way that will apply across platforms and does not require modification of the children objects.  Instead of calculating the address of the saved instruction pointer, which might be different for different platforms.  I'm using `info frame` now to get the saved instruction pointer location.  It's also the register $fp now for the frame pointer, which gdb says is generic across platforms like just like the $sp register.

I will test it out on some of the platforms to verify it works before submitting a new version. Again, thanks for the comments.

Reply:
Yep that sounds like a better way to do it.